rules/aip0192/aip0192.go

```diff
@@ -35,6 +38,22 @@ func AddRules(r lint.RuleRegistry) error {
 	)
 }
 
+// Returns true if this is a deprecated method, false otherwise.
+func isDeprecatedMethod(m *desc.MethodDescriptor) bool {
+	if m.GetMethodOptions() == nil {
+		return false
+	}
+	return *m.GetMethodOptions().Deprecated
```

Comment:
IDK off the top of my head, but is there a `GetDeprecated` accessor? If so, please use that instead of direct field access.

Reply:
Yeah, I think we just want:

```go
return m.GetMethodOptions().GetDeprecated()
```

Reply:
There doesn't seem to be one.